.github/workflows/proto.yml

```diff
@@ -1,13 +1,30 @@
-name: buf
+name: Proto
 on:
   pull_request:
   push:
     branches:
         - main
 jobs:
-  build-lint-push-protos:
+  changes:
```

Comment:
Nit: `protos-changed:`?

Reply:
It's all under the "buf" job named "protos"?